openmc/tallies.py

```diff
@@ -1078,8 +1070,8 @@ def contains_filter(self, filter_type):
 
         Parameters
         ----------
-        filter_type : str
-            Type of the filter, e.g. 'mesh'
+        filter_type : openmc.filter.FilterMeta
```

Comment:
These methods want a class, not an instance of a class.  So listing this docstring as `openmc.Filter` would imply that
```python
f = openmc.EnergyFilter(bins)
tally.contains_filter(f)
```
would work, but it will not.  Instead, it should be called
```python
tally.contains_filter(openmc.EnergyFilter)
```